R/prep_alignment_table.R

```diff
@@ -90,7 +91,10 @@ prep_alignment_table <- function(results_portfolio,
         green_brown = "green_or_brown"
       ) %>%
       filter(.data$asset_class == .env$asset_class)
-  }
+    } else {
+      data_out <- tibble::tibble()
```

Comment:
NIT: It might be preferable to return an empty tibble with the expected column names/ structure of real output, rather than just an empty tibble. This would allow the function to continue to work in pipes, where downstream functions require named data-frames.